lib/ansible/content.rb

```diff
@@ -0,0 +1,17 @@
+module Ansible
+  class Content
+    attr_accessor :path
+
+    def initialize(path)
+      @path = path
+    end
+
+    def fetch_galaxy_roles
+      roles_path = path.join('roles')
+      role_file  = path.join('requirements.yml')
+
+      params = ["install", :roles_path= => roles_path, :role_file= => role_file]
```

Comment:
I don't think you need the trailing `=`

Reply:
Same as above, this is how it was originally.